### PR TITLE
Document data-loss bug with invalid expires dates

### PR DESCRIPTION
**Pull request by dfabulich**
See also #57 and #162.

### CODE REVIEW

File: explainer.md [hunk 1]
```diff
@@ -527,6 +527,38 @@ JavaScript Date objects will be automatically converted to timestamps thanks to
 
 Tutorials on this feature might want to cover the following topics.
 
+### Data loss with invalid expires timestamps
+
+**The expiration date is an absolute timestamp, not a relative time interval.** If users of this API pass in a small non-zero number for `expires`, e.g. `expires: 30` or `expires: 1000`, the Cookie Store API will delete the cookie data, resulting in data loss.
```
Comment (ayuishii) [inline explainer.md:532]: Can we remove the bold here?

**Review by ayuishii — COMMENTED** [review]
Thanks for adding this @dfabulich!
Can you join the [WICG](https://www.w3.org/community/wicg/join) so we can add this change?

**Comment by dfabulich** [comment]
I've finally joined WICG, with approval from my employer. Let's merge this.

**Comment by inexorabletash** [comment]
Ugh, sorry for letting this linger. Looking...

**Comment by inexorabletash** [comment]
@dfabulich - Can you follow the instructions on https://labs.w3.org/repo-manager/pr/id/WICG/cookie-store/163 to "they need to link their GitHub account with a W3C account." [link](https://www.w3.org/users/myprofile/connectedaccounts) - note that the IPR bot was having issues, though...

**Pull request merged**